Oneirocom/Magick
# Posthog tracking

## What Changed:

- Adds posthog custom event tracking support on agents and app. Events sent out on discord message and response.  Also tracks the ful event metadata besides sensitive content.

## How to test:

- Make sure you have the appropriate serverside key - POSTHOG_API_KEY
- Chat with a discord bot
- Check your posthog environment for events

## Additional information:

Any other information that might be useful while reviewing.
